app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -19,5 +19,4 @@
 }
 
 $govuk-assets-path: '/customs-movements/assets/';
-$govuk-images-path: "/customs-movements/assets/lib/govuk-frontend/govuk/assets/images/";
-$govuk-fonts-path: "/customs-movements/assets/lib/govuk-frontend/govuk/assets/fonts/";
+$govuk-fonts-path: "/customs-movements/assets/lib/govuk-frontend/govuk/assets/";
```

Comment:
Just checking - the fonts don't live in /fonts then?

Reply:
I think I was experimenting and forgot about it. thanks for that.